src/main/java/org/folio/rest/impl/TenantReferenceAPI.java

```diff
@@ -43,10 +46,10 @@ public Future<Integer> loadData(TenantAttributes attributes, String tenantId, Ma
     buildDataLoadingParameters(attributes, tl);
 
     return Future.succeededFuture()
+      // migrationModule value it the same as fromModuleVersion from schema.json
+      .compose(v -> migration(attributes, "mod-invoice-storage-5.7.0", () -> populateInvoiceWithFiscalYear(headers, vertxContext)))
```

Comment:
if migrationModule is 'migrationTo' - it should be "mod-invoice-storage-5.8.0" - because this version is the next version to release, it is Quesnelia release version

Reply:
based on current logic it is "from", I will change to "moduleTo" to avoid consufion
![image](https://github.com/folio-org/mod-invoice-storage/assets/113523904/b8a8c7b3-758b-426a-a75c-bac33719808f)
my comment: https://github.com/folio-org/mod-invoice-storage/pull/171#discussion_r1508921040

Reply:
kept previous approach and changed to newer version